### PR TITLE
Change to allow the use of matching skill colors in XP Globes

### PR DESCRIPTION
**Pull request by Teedjee**
Added an option to use colors matching to skills in the progressbar of the skill globes instead of one color to use for all the globes.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java [hunk 1]
```diff
@@ -47,11 +47,22 @@ default boolean enableTooltips()
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "enableSkillColors",
+			name = "Enable skill colors",
+			description = "Use colors matching to the skill",
+			position = 1
+	)
+	default boolean enableSkillColors()
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java]: Why is this even toggle?

Comment (Teedjee) [reply runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java]: I didn't want to remove the option to set a value for all orbs.
With this change it's default on skill colors but you can uncheck the option and then set 
a value yourself.

**Comment by SoyChai** [comment]
Conflicting opinion here: I think we actually *should* have a toggle for this, since any user who currently uses their own custom orb colours will see this as a feature removal.

**Comment by deathbeam** [comment]
I would just go with how OSB does it, do they have toggle?

**Pull request closed without merging**